### ISSUE TITLE
Where is the gamma in the code implemetion?

### ISSUE THREAD

**Issue by Diracckjy** [litian96/FedProx#39]
According to Algorithm 2, there is a parameter gamma in the input which measures how much local
computation is performed to solve the local subproblem on device k at the t-th round.
![image](https://github.com/litian96/FedProx/assets/45199852/e1c2a974-1a57-496d-9b25-a691f0251ebd)
But I can't find gamma in the code implemention. 
In (https://github.com/litian96/FedProx/blob/master/flearn/models/mnist/mclr.py) there is only a variable num_epochs.
def solve_inner(self, data, num_epochs=1, batch_size=32):
        '''Solves local optimization problem'''
        for _ in trange(num_epochs, desc='Epoch: ', leave=False, ncols=120):
            for X, y in batch_data(data, batch_size):
                with self.graph.as_default():
                    self.sess.run(self.train_op,
                        feed_dict={self.features: X, self.labels: y})
        soln = self.get_params()
        comp = num_epochs * (len(data['y'])//batch_size) * batch_size * self.flops
        return soln, comp
So could please help me find gamma?

**Comment by andzhou** [comment]
yes， I also can't find the gamma
